src/main/java/uk/gov/hmcts/juror/api/moj/service/trial/PanelServiceImpl.java

```diff
@@ -135,7 +135,7 @@ private void addPanelMembersTrialValidationChecks(String trialNumber,
         Optional<Trial> trial = trialRepository.findByTrialNumberAndCourtLocationLocCode(trialNumber,
             courtLocationCode);
 
-        if (trial.isEmpty()) {
+        if (!trial.isPresent()) {
```

Comment:
does this need to be a negative check? can it remain as isEmpty (for improved readability?